src/main/java/com/example/WorldOfAnimals/controllers/AnimalBreedController.java

```diff
@@ -18,31 +21,53 @@
 
 import java.util.List;
 
+@Tag(
+        name = "Animal breeds",
+        description = "Manage animal breeds by define " +
+                "(existed in base) animal type"
+)
 @RestController
 @RequestMapping("/animals/breeds")
 @AllArgsConstructor
 public class AnimalBreedController {
 
     private AnimalBreedService service;
 
+    @Operation(
```

Comment:
Maybe need think about this code 
`    @ApiResponses({
            @ApiResponse(responseCode = "200"),
            @ApiResponse(responseCode = "500")
    })`

Reply:
This is for all controllers

Reply:
I added one exapmle for checking. But I afraid that code will be overload for reading if I will add @ApiResponses to all controllers.

Reply:
How will the code be overloaded? 
Why you use Swagger if it doesn't help you call API and read API documentation ?

Reply:
done